src/Sylius/Bundle/CoreBundle/Migrations/Version20171003103916.php

```diff
@@ -37,7 +37,18 @@ public function up(Schema $schema): void
         $defaultLocale = $this->container->getParameter('locale');
         $productAttributeRepository = $this->container->get('sylius.repository.product_attribute');
 
-        $productAttributes = $productAttributeRepository->findBy(['type' => SelectAttributeType::TYPE]);
+        $productAttributes = $productAttributeRepository
```

Comment:
The existing migrations shouldn't be changed

Reply:
The problem is, this migration is blocking changes to the attribute mapping as described in: https://github.com/Sylius/Sylius/issues/11592

Reply:
https://github.com/Sylius/Sylius/pull/11624 was actually supposed to have fixed that.

Reply:
This is now fixed in the master